### ISSUE TITLE
Allow to promote a (set of) package

### ISSUE THREAD

**Issue by Drup** [dsheets/codoc#72]
The current behavior of `opam doc` is very nice if you want to build the doc for a whole snapshot of opam, but not if you want to build the doc for only a small set of packages. `opam doc` should figure out the minimal transitive dependences for me and promote the given package in the first page.

**Comment by dsheets** [comment]
This sounds like a good feature. What do you mean by "promote"? Do you want all the docs built but only some of them indexed?

**Comment by Drup** [comment]
Yes. A bit like you can write an indexdoc for ocamldoc to show only the modules you want, but at the package level (On that note, a file where I can put markup and use as front page would be nice too).

To take an example, in the documentation for llvmgraph, I want the main index to be directly the Llvmgraph module even if ocamlgraph's and LLvm's modules are accessible by crosslinks.
